### ISSUE TITLE
Investigating validitiy of lick interval warnings

### ISSUE THREAD

**Issue by micahwoodard** [AllenNeuralDynamics/dynamic-foraging-task#1059]
After merging in calculating lick interval percentages, there have been more than expected instances of same side and cros side licks over 10%. We need to investigate whether this is an issue on scopes or with code.

Cross side issues per box: 
- ephys1A: 2
- - ephys3A: 1
- ephys4A: 1
- 2B: 1
- 3C: 1
- 3D: 1
- 6A: 2
- 6B: 1
- 7A: 1
- 7C: 1
- 7D: 2
- 8D: 1
- 7B: 1

Same side issues per box: 
- ephys3A: 5
- 1A: 1
- 1D: 1
- 6A: 1
- 7B: 1
- 7C: 1

**Comment by ellahiltonvano** [comment]
Updated list of issues per box (posted week of 11/25/2024):

**Cross side issues per box:** 
ephys1: 5
ephys3: 1
ephys4: 2
2B: 1
3C: 1
3D: 1
6A: 3
6B: 3
6D: 2
7A: 2
7B: 3
7C: 7
7D: 5
8A: 1
8C: 1
8D: 2
9D: 2

**Same side issues per box:** 
ephys3: 13
ephys4: 1
1A: 1
1C: 1
1D: 1
6A: 2
7B: 2
7C: 1

**Comment by alexpiet** [comment]
@ZhixiaoSu recommends changing the definition of cross-side licks
https://github.com/AllenNeuralDynamics/dynamic-foraging-task/issues/1026#issuecomment-2513330895

@JeremiahYCohen suggests decreasing 100ms to 50ms

**Comment by ellahiltonvano** [comment]
**Updated 12/23/24:** 

**Cross side issues per box:** 
- 1B: 2
- 1C: 1
- 1D: 2
- 3C: 3
- 3D: 1
- 6A: 2
- 6B: 3
- 6D: 3
- 7A: 2
- 7B: 5
- 7C: 4
- 7D: 6
- 8A: 2
- 8B: 2
- 8C: 3
- 8D: 2
- ephys1: 5
- ephys3: 5
- ephys4: 3
- 9D: 2

**Same side issues per box:** 
- 1A: 3
- 1C: 3
- 1D: 3
- 2B: 1
- 3C: 2
- 6A: 2
- 6D: 2
- 7B: 3
- 7C: 1
- 8B: 1
- ephys3: 15
- ephys4: 2

**Comment by XX-Yin** [comment]
This appears to be a very serious issue and might worth a discussion in a separate place.

While some of the lick detection problems could potentially be mitigated by adjusting parameters and the sensitivity of the lick detector, I’ve observed that the Janelia lick detector often lacks stability and consistency across different mice and days. This seems to be an inherent hardware limitation of the device.

It might be worth discussing with SIPE the possibility of developing a more stable lick detector specifically for the behavior rig. It's ok if it has licking artefacts during ephys recording when we lack an idea solution, which could be a separate issue.

**Comment by ellahiltonvano** [comment]
@XX-Yin I agree we need to discuss more. We are waiting on some feedback from Sue and will likely discuss in a meeting in January.

**Comment by ellahiltonvano** [comment]
@alexpiet previously assigned to Micah, assuming we'll close this after it is part of QC

**Comment by alexpiet** [comment]
Part of QC now
